packages/cli-platform-android/src/config/index.ts

```diff
@@ -132,10 +132,12 @@ export function dependencyConfig(
   const androidMkPath = userConfig.androidMkPath
     ? path.join(sourceDir, userConfig.androidMkPath)
     : path.join(sourceDir, 'build/generated/source/codegen/jni/Android.mk');
-  const cmakeListsPath = userConfig.cmakeListsPath
+  let cmakeListsPath = userConfig.cmakeListsPath
     ? path.join(sourceDir, userConfig.cmakeListsPath)
     : path.join(sourceDir, 'build/generated/source/codegen/jni/CMakeLists.txt');
-
+  if(process.platform == "win32") {
```

Comment:
Let's use strict comparison:
```suggestion
  if (process.platform === "win32") {
```
Also, can you run `yarn lint --fix` to make sure the formatting is done by Prettier? :)